### ISSUE TITLE
Maintenance release Greenhouse Effect 1.0 prototype with improved temperatures and readouts

### ISSUE THREAD

**Issue by jbphet** [phetsims/greenhouse-effect#112]
In https://github.com/phetsims/greenhouse-effect/issues/98 work was done that added a decimal place to the temperature readouts and adjusted the temperatures for the dates to correspond more closely to the historical record.  We reviewed these changes in our weekly design meeting on 11/12/2021, and @kathy-phet requested that once these are finalized, I should do a maintenance release of the 1.0 prototype version with these improvements.

I'll do this once we've fully approved and closed #98.

**Comment by jbphet** [comment]
A couple of things to note about how this went:
- I had to make the changes manually, since the sim had been converted to TypeScript before these changes were made on master.
- For some reason, the gas concentration values that were experimentally determined in #98 didn't quite work, and I had to determine them again for this maintenance release.  They had to be slightly higher in order to get the same temperature value.
